packages/core/src/components/sections/ProductDetails/DefaultComponents.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  ProductTitle as UIProductTitle,
+  DiscountBadge as UIDiscountBadge,
+  BuyButton as UIBuyButton,
+  SkuSelector as UISkuSelector,
+  ShippingSimulation as UIShippingSimulation,
+  Icon as UIIcon,
+  Price as UIPrice,
+  QuantitySelector as UIQuantitySelector,
+  ImageGalleryViewer as UIImageGalleryViewer,
+  ImageGallery as UIImageGallery,
+} from '@faststore/ui'
+
+import LocalImageGallery from 'src/components/ui/ImageGallery'
+import LocalShippingSimulation from 'src/components/ui/ShippingSimulation/ShippingSimulation'
+import { Image } from 'src/components/ui/Image'
+import LocalNotAvailableButton from 'src/components/product/NotAvailableButton'
+
+export const ProductDetailsDefaultComponents = {
+  ProductTitle: UIProductTitle,
+  DiscountBadge: UIDiscountBadge,
+  BuyButton: UIBuyButton,
+  Icon: UIIcon,
+  Price: UIPrice,
+  QuantitySelector: UIQuantitySelector,
+  SkuSelector: UISkuSelector,
+  ShippingSimulation: UIShippingSimulation,
+  ImageGallery: UIImageGallery,
+  ImageGalleryViewer: UIImageGalleryViewer,
+  __experimentalImageGalleryImage: Image,
+  __experimentalImageGallery: LocalImageGallery,
```

Comment:
Why are we using `Local` prefix for those core ui components (but Image)?

Reply:
It was arbitrary. Would you suggest something different?

Reply:
oh, it's just because we weren't using this prefix before, so I was unsure if there was any additional purpose 😕

Reply:
We were! In the Overrides file.